dotCMS/src/main/java/com/dotmarketing/business/cache/provider/hazelcast/AbstractHazelcastCacheProvider.java

```diff
@@ -107,7 +114,12 @@ public void remove(String group, String key) {
             return;
         }
         try{
-            getHazelcastInstance().getMap(group).remove(key);
+            if(ASYNC_PUT){
+                getHazelcastInstance().getMap(group).removeAsync(key);
+            }
+            else{
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "else" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)